src/vs/workbench/contrib/terminal/common/terminalConfiguration.ts

```diff
@@ -606,6 +606,11 @@ const terminalConfiguration: IConfigurationNode = {
 			],
 			default: 'both'
 		},
+		[TerminalSettingId.ShellIntegrationEnvironmentReporting]: {
+			markdownDescription: localize('terminal.integrated.shellIntegration.environmentReporting', "Controls whether to use terminal environment API to report environment state to VS Code"),
```

Comment:
```suggestion
			markdownDescription: localize('terminal.integrated.shellIntegration.environmentReporting', "Controls whether to report the shell environment, enabling its use in features such as {0}. This may cause a slowdown when printing your shell's prompt.", `\`#${TerminalContribSettingId.SuggestEnabled}#\``),
```

Reply:
For that suggestion you'll need to also import `TerminalContribSettingId` to get it compiling